### DISCUSSION TITLE
plasma:grey-grey generates greyscale image

### DISCUSSION THREAD

**Discussion by pgajdos** [ImageMagick/ImageMagick#2409] [General]
Hi,

unlike in [examples of plasma usage](http://www.imagemagick.org/Usage/canvas/#plasma),
```
magick convert -size 100x100  plasma:grey-grey            plasma_grey.jpg
```
produces greyscale image. It is true for both 6.9.11-27 and 7.0.10-27. Is that now considered correct result?

**Comment by urban-warrior** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in the GIT master branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ https://www.imagemagick.org/download/beta/ by sometime tomorrow.

**Comment by pgajdos** [comment]
That helped, thanks a lot!
